docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+# For running container dependencies locally
```

Comment:
Needed a Redis instance locally and this seemed easier than adjusting the included Dockerfile.